Test/Tests/RealtimeClientChannelTests.swift

```diff
@@ -3791,10 +3721,13 @@ class RealtimeClientChannelTests: XCTestCase {
             attachedMessageWithError.action = .attached
             attachedMessageWithError.channel = channel.name
 
-            channel.once(.update) { stateChange in
+            channel.on { stateChange in
```

Comment:
Why this change from `once` to `on`?

Reply:
I think it should be `on`, bc in case of invalid event the test will fail (on event assertion), which will signal about improper library behavior.

Reply:
OK, makes sense.